### PR TITLE
WINDUP-516: Multiple valueStores even if the rule does not use parameter...

### PR DESCRIPTION
**Pull request by mbriskar**
...s

### CODE REVIEW

File: rules-java-ee/addon/src/main/resources/legacy/org.jboss.windup.rules.apps.legacy.java.JBossConfig.windup.xml [hunk 1]
```diff
@@ -0,0 +1,95 @@
+<?xml version="1.0"?>
```
Comment (jsight) [inline rules-java-ee/addon/src/main/resources/legacy/org.jboss.windup.rules.apps.legacy.java.JBossConfig.windup.xml]: This file should not be here. If changes are made, they should be made to the windup-rulesets repository.

**Comment by lincolnthree** [comment]
I think this same bug might happen in the JavaClass condition as well. Wdyt?

**Comment by windupgithubbot1** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://10.3.8.5//job/windup-pr-builder/516/<h2>Failed Tests: <span class='status-failure'>13</span></h2><h3><a name='org.jboss.windup.rules.apps:windup-rules-java-ee-tests' /><a href='http://10.3.8.5/job/windup-pr-builder/516/org.jboss.windup.rules.apps$windup-rules-java-ee-tests/testReport'>org.jboss.windup.rules.apps:windup-rules-java-ee-tests</a>: <span class='status-failure'>1</span></h3><ul><li><a href='http://10.3.8.5/job/windup-pr-builder/516/org.jboss.windup.rules.apps$windup-rules-java-ee-tests/testReport/org.jboss.windup.rules.apps.javaee.tests/DiscoverEjbConfigurationTest/testEJBMetadataExtraction/'><strong>org.jboss.windup.rules.apps.javaee.tests.DiscoverEjbConfigurationTest.testEJBMetadataExtraction</strong></a></li></ul><h3><a name='org.jboss.windup.rules.apps:windup-rules-xml-tests' /><a href='http://10.3.8.5/job/windup-pr-builder/516/org.jboss.windup.rules.apps$windup-rules-xml-tests/testReport'>org.jboss.windup.rules.apps:windup-rules-xml-tests</a>: <span class='status-failure'>4</span></h3><ul><li><a href='http://10.3.8.5/job/windup-pr-builder/516/org.jboss.windup.rules.apps$windup-rules-xml-tests/testReport/org.jboss.windup.rules.xml/XmlFileParameterizedTest/testXmlParams/'><strong>org.jboss.windup.rules.xml.XmlFileParameterizedTest.testXmlParams</strong></a></li><li><a href='http://10.3.8.5/job/windup-pr-builder/516/org.jboss.windup.rules.apps$windup-rules-xml-tests/testReport/org.jboss.windup.rules.xml/XmlFileParameterizedTest/testXmlParamsDangling/'><strong>org.jboss.windup.rules.xml.XmlFileParameterizedTest.testXmlParamsDangling</strong></a></li><li><a href='http://10.3.8.5/job/windup-pr-builder/516/org.jboss.windup.rules.apps$windup-rules-xml-tests/testReport/org.jboss.windup.rules.xml/XmlFileParameterizedTest/testXmlParamsMultiCondition/'><strong>org.jboss.windup.rules.xml.XmlFileParameterizedTest.testXmlParamsMultiCondition</strong></a></li><li><a href='http://10.3.8.5/job/windup-pr-builder/516/org.jboss.windup.rules.apps$windup-rules-xml-tests/testReport/org.jboss.windup.rules.xml.legacy/XmlJonasConfigTest/testHintsAndClassificationOperation/'><strong>org.jboss.windup.rules.xml.legacy.XmlJonasConfigTest.testHintsAndClassificationOperation</strong></a></li></ul><h3><a name='org.jboss.windup.tests:windup-tests' /><a href='http://10.3.8.5/job/windup-pr-builder/516/org.jboss.windup.tests$windup-tests/testReport'>org.jboss.windup.tests:windup-tests</a>: <span class='status-failure'>8</span></h3><ul><li><a href='http://10.3.8.5/job/windup-pr-builder/516/org.jboss.windup.tests$windup-tests/testReport/org.jboss.windup.tests.application/WindupArchitectureHibernateTest/testRunWindupHibernate/'><strong>org.jboss.windup.tests.application.WindupArchitectureHibernateTest.testRunWindupHibernate</strong></a></li><li><a href='http://10.3.8.5/job/windup-pr-builder/516/org.jboss.windup.tests$windup-tests/testReport/org.jboss.windup.tests.application/WindupArchitectureJEEExampleTest/testRunWindupJEEExampleMode/'><strong>org.jboss.windup.tests.application.WindupArchitectureJEEExampleTest.testRunWindupJEEExampleMode</strong></a></li><li><a href='http://10.3.8.5/job/windup-pr-builder/516/org.jboss.windup.tests$windup-tests/testReport/org.jboss.windup.tests.application/WindupArchitectureMediumBinaryModeTest/testRunWindupMedium/'><strong>org.jboss.windup.tests.application.WindupArchitectureMediumBinaryModeTest.testRunWindupMedium</strong></a></li><li><a href='http://10.3.8.5/job/windup-pr-builder/516/org.jboss.windup.tests$windup-tests/testReport/org.jboss.windup.tests.application/WindupArchitectureSeamBookingSourceTest/testRunWindupSeamBookingSourceMode/'><strong>org.jboss.windup.tests.application.WindupArchitectureSeamBookingSourceTest.testRunWindupSeamBookingSourceMode</strong></a></li><li><a href='http://10.3.8.5/job/windup-pr-builder/516/org.jboss.windup.tests$windup-tests/testReport/org.jboss.windup.tests.application/WindupArchitectureSmallBinaryModeTest/testRunWindupTiny/'><strong>org.jboss.windup.tests.application.WindupArchitectureSmallBinaryModeTest.testRunWindupTiny</strong></a></li><li><a href='http://10.3.8.5/job/windup-pr-builder/516/org.jboss.windup.tests$windup-tests/testReport/org.jboss.windup.tests.application/WindupArchitectureSoa5GatewaysRoutersTest/testRunWindupSourceMode/'><strong>org.jboss.windup.tests.application.WindupArchitectureSoa5GatewaysRoutersTest.testRunWindupSourceMode</strong></a></li><li><a href='http://10.3.8.5/job/windup-pr-builder/516/org.jboss.windup.tests$windup-tests/testReport/org.jboss.windup.tests.application/WindupArchitectureSourceModeTest/testRunWindupSourceMode/'><strong>org.jboss.windup.tests.application.WindupArchitectureSourceModeTest.testRunWindupSourceMode</strong></a></li><li><a href='http://10.3.8.5/job/windup-pr-builder/516/org.jboss.windup.tests$windup-tests/testReport/org.jboss.windup.tests.application/WindupArchitectureSpringSmallTest/testRunWindupSmallSpringApp/'><strong>org.jboss.windup.tests.application.WindupArchitectureSpringSmallTest.testRunWindupSmallSpringApp</strong></a></li></ul>
Test FAILed.

**Comment by windupgithubbot1** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://10.3.8.5//job/windup-pr-builder/517/
Test FAILed.

**Comment by windupgithubbot1** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://10.3.8.5//job/windup-pr-builder/518/<h2>Failed Tests: <span class='status-failure'>3</span></h2><h3><a name='org.jboss.windup.rules.apps:windup-rules-xml-tests' /><a href='http://10.3.8.5/job/windup-pr-builder/518/org.jboss.windup.rules.apps$windup-rules-xml-tests/testReport'>org.jboss.windup.rules.apps:windup-rules-xml-tests</a>: <span class='status-failure'>3</span></h3><ul><li><a href='http://10.3.8.5/job/windup-pr-builder/518/org.jboss.windup.rules.apps$windup-rules-xml-tests/testReport/org.jboss.windup.rules.xml/XmlFileParameterizedTest/testXmlParams/'><strong>org.jboss.windup.rules.xml.XmlFileParameterizedTest.testXmlParams</strong></a></li><li><a href='http://10.3.8.5/job/windup-pr-builder/518/org.jboss.windup.rules.apps$windup-rules-xml-tests/testReport/org.jboss.windup.rules.xml/XmlFileParameterizedTest/testXmlParamsDangling/'><strong>org.jboss.windup.rules.xml.XmlFileParameterizedTest.testXmlParamsDangling</strong></a></li><li><a href='http://10.3.8.5/job/windup-pr-builder/518/org.jboss.windup.rules.apps$windup-rules-xml-tests/testReport/org.jboss.windup.rules.xml/XmlFileParameterizedTest/testXmlParamsMultiCondition/'><strong>org.jboss.windup.rules.xml.XmlFileParameterizedTest.testXmlParamsMultiCondition</strong></a></li></ul>
Test FAILed.

**Comment by windupgithubbot1** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://10.3.8.5//job/windup-pr-builder/519/<h2>Failed Tests: <span class='status-failure'>3</span></h2><h3><a name='org.jboss.windup.rules.apps:windup-rules-xml-tests' /><a href='http://10.3.8.5/job/windup-pr-builder/519/org.jboss.windup.rules.apps$windup-rules-xml-tests/testReport'>org.jboss.windup.rules.apps:windup-rules-xml-tests</a>: <span class='status-failure'>3</span></h3><ul><li><a href='http://10.3.8.5/job/windup-pr-builder/519/org.jboss.windup.rules.apps$windup-rules-xml-tests/testReport/org.jboss.windup.rules.xml/XmlFileParameterizedTest/testXmlParams/'><strong>org.jboss.windup.rules.xml.XmlFileParameterizedTest.testXmlParams</strong></a></li><li><a href='http://10.3.8.5/job/windup-pr-builder/519/org.jboss.windup.rules.apps$windup-rules-xml-tests/testReport/org.jboss.windup.rules.xml/XmlFileParameterizedTest/testXmlParamsDangling/'><strong>org.jboss.windup.rules.xml.XmlFileParameterizedTest.testXmlParamsDangling</strong></a></li><li><a href='http://10.3.8.5/job/windup-pr-builder/519/org.jboss.windup.rules.apps$windup-rules-xml-tests/testReport/org.jboss.windup.rules.xml/XmlFileParameterizedTest/testXmlParamsMultiCondition/'><strong>org.jboss.windup.rules.xml.XmlFileParameterizedTest.testXmlParamsMultiCondition</strong></a></li></ul>
Test FAILed.

**Comment by windupgithubbot1** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://10.3.8.5//job/windup-pr-builder/520/<h2>Failed Tests: <span class='status-failure'>3</span></h2><h3><a name='org.jboss.windup.rules.apps:windup-rules-xml-tests' /><a href='http://10.3.8.5/job/windup-pr-builder/520/org.jboss.windup.rules.apps$windup-rules-xml-tests/testReport'>org.jboss.windup.rules.apps:windup-rules-xml-tests</a>: <span class='status-failure'>3</span></h3><ul><li><a href='http://10.3.8.5/job/windup-pr-builder/520/org.jboss.windup.rules.apps$windup-rules-xml-tests/testReport/org.jboss.windup.rules.xml/XmlFileParameterizedTest/testXmlParams/'><strong>org.jboss.windup.rules.xml.XmlFileParameterizedTest.testXmlParams</strong></a></li><li><a href='http://10.3.8.5/job/windup-pr-builder/520/org.jboss.windup.rules.apps$windup-rules-xml-tests/testReport/org.jboss.windup.rules.xml/XmlFileParameterizedTest/testXmlParamsDangling/'><strong>org.jboss.windup.rules.xml.XmlFileParameterizedTest.testXmlParamsDangling</strong></a></li><li><a href='http://10.3.8.5/job/windup-pr-builder/520/org.jboss.windup.rules.apps$windup-rules-xml-tests/testReport/org.jboss.windup.rules.xml/XmlFileParameterizedTest/testXmlParamsMultiCondition/'><strong>org.jboss.windup.rules.xml.XmlFileParameterizedTest.testXmlParamsMultiCondition</strong></a></li></ul>
Test FAILed.

**Comment by mbriskar** [comment]
woops, will fix that on Monday

**Comment by mbriskar** [comment]
The problem with JavaClasses is they work with different principle. XmlFile.from() operates with the files that matched previous xpath. JavaClass.from() operates only with JavaTypeReferenceModel's that matched previous JavaClass condition. These are just lines and their place in the actual file doesn't mean that much.

**Comment by mbriskar** [comment]
retest this please

**Comment by windupgithubbot1** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://10.3.8.5//job/windup-pr-builder/522/
Test PASSed.

**Comment by windupgithubbot1** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://10.3.8.5//job/windup-pr-builder/523/
Test PASSed.

**Comment by windupgithubbot1** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://10.3.8.5//job/windup-pr-builder/524/
Test PASSed.

**Comment by windupgithubbot1** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://10.3.8.5//job/windup-pr-builder/525/<h2>Failed Tests: <span class='status-failure'>28</span></h2><h3><a name='org.jboss.windup.config:windup-config-tests' /><a href='http://10.3.8.5/job/windup-pr-builder/525/org.jboss.windup.config$windup-config-tests/testReport'>org.jboss.windup.config:windup-config-tests</a>: <span class='status-failure'>4</span></h3><ul><li><a href='http://10.3.8.5/job/windup-pr-builder/525/org.jboss.windup.config$windup-config-tests/testReport/org.jboss.windup.config.parameters/ParameterWiringTest/testIterationVariableResolving/'><strong>org.jboss.windup.config.parameters.ParameterWiringTest.testIterationVariableResolving</strong></a></li><li><a href='http://10.3.8.5/job/windup-pr-builder/525/org.jboss.windup.config$windup-config-tests/testReport/org.jboss.windup.config.parameters/ParameterWiringTest/testIterationVariableResolving2/'><strong>org.jboss.windup.config.parameters.ParameterWiringTest.testIterationVariableResolving2</strong></a></li><li><a href='http://10.3.8.5/job/windup-pr-builder/525/org.jboss.windup.config$windup-config-tests/testReport/org.jboss.windup.config.parameters/ParameterWiringTest/testIterationVariableResolving3/'><strong>org.jboss.windup.config.parameters.ParameterWiringTest.testIterationVariableResolving3</strong></a></li><li><a href='http://10.3.8.5/job/windup-pr-builder/525/org.jboss.windup.config$windup-config-tests/testReport/org.jboss.windup.config.parameters/ParameterWiringTest/testIterationVariableResolving4/'><strong>org.jboss.windup.config.parameters.ParameterWiringTest.testIterationVariableResolving4</strong></a></li></ul><h3><a name='org.jboss.windup.rules.apps:windup-rules-base-tests' /><a href='http://10.3.8.5/job/windup-pr-builder/525/org.jboss.windup.rules.apps$windup-rules-base-tests/testReport'>org.jboss.windup.rules.apps:windup-rules-base-tests</a>: <span class='status-failure'>1</span></h3><ul><li><a href='http://10.3.8.5/job/windup-pr-builder/525/org.jboss.windup.rules.apps$windup-rules-base-tests/testReport/org.jboss.windup.rules.apps.condition/FileContentTest/testFileContentScan/'><strong>org.jboss.windup.rules.apps.condition.FileContentTest.testFileContentScan</strong></a></li></ul><h3><a name='org.jboss.windup.rules.apps:windup-rules-java-tests' /><a href='http://10.3.8.5/job/windup-pr-builder/525/org.jboss.windup.rules.apps$windup-rules-java-tests/testReport'>org.jboss.windup.rules.apps:windup-rules-java-tests</a>: <span class='status-failure'>7</span></h3><ul><li><a href='http://10.3.8.5/job/windup-pr-builder/525/org.jboss.windup.rules.apps$windup-rules-java-tests/testReport/org.jboss.windup.rule.groovy/GroovyExtensionJavaRulesTest/testHintsAndClassificationOperation/'><strong>org.jboss.windup.rule.groovy.GroovyExtensionJavaRulesTest.testHintsAndClassificationOperation</strong></a></li><li><a href='http://10.3.8.5/job/windup-pr-builder/525/org.jboss.windup.rules.apps$windup-rules-java-tests/testReport/org.jboss.windup.rules.java/GatherIgnoredFilesTest/testJavaClassCondition/'><strong>org.jboss.windup.rules.java.GatherIgnoredFilesTest.testJavaClassCondition</strong></a></li><li><a href='http://10.3.8.5/job/windup-pr-builder/525/org.jboss.windup.rules.apps$windup-rules-java-tests/testReport/org.jboss.windup.rules.java/JavaClassCompositeTest/testJavaClassCondition/'><strong>org.jboss.windup.rules.java.JavaClassCompositeTest.testJavaClassCondition</strong></a></li><li><a href='http://10.3.8.5/job/windup-pr-builder/525/org.jboss.windup.rules.apps$windup-rules-java-tests/testReport/org.jboss.windup.rules.java/JavaClassTest/testJavaClassCondition/'><strong>org.jboss.windup.rules.java.JavaClassTest.testJavaClassCondition</strong></a></li><li><a href='http://10.3.8.5/job/windup-pr-builder/525/org.jboss.windup.rules.apps$windup-rules-java-tests/testReport/org.jboss.windup.rules.java/JavaClassXmlRulesTest/testJavaClassCondition/'><strong>org.jboss.windup.rules.java.JavaClassXmlRulesTest.testJavaClassCondition</strong></a></li><li><a href='http://10.3.8.5/job/windup-pr-builder/525/org.jboss.windup.rules.apps$windup-rules-java-tests/testReport/org.jboss.windup.rules.java/JavaHintsClassificationsTest/testHintsAndClassificationOperation/'><strong>org.jboss.windup.rules.java.JavaHintsClassificationsTest.testHintsAndClassificationOperation</strong></a></li><li><a href='http://10.3.8.5/job/windup-pr-builder/525/org.jboss.windup.rules.apps$windup-rules-java-tests/testReport/org.jboss.windup.rules.java.ignore/JavaIgnoreRegexesTest/testHintsAndClassificationOperation/'><strong>org.jboss.windup.rules.java.ignore.JavaIgnoreRegexesTest.testHintsAndClassificationOperation</strong></a></li></ul><h3><a name='org.jboss.windup.rules.apps:windup-rules-xml-tests' /><a href='http://10.3.8.5/job/windup-pr-builder/525/org.jboss.windup.rules.apps$windup-rules-xml-tests/testReport'>org.jboss.windup.rules.apps:windup-rules-xml-tests</a>: <span class='status-failure'>9</span></h3><ul><li><a href='http://10.3.8.5/job/windup-pr-builder/525/org.jboss.windup.rules.apps$windup-rules-xml-tests/testReport/org.jboss.windup.rules.apps.xml.groovy/GroovyExtensionXmlRulesTest/testHintsAndClassificationOperation/'><strong>org.jboss.windup.rules.apps.xml.groovy.GroovyExtensionXmlRulesTest.testHintsAndClassificationOperation</strong></a></li><li><a href='http://10.3.8.5/job/windup-pr-builder/525/org.jboss.windup.rules.apps$windup-rules-xml-tests/testReport/org.jboss.windup.rules.xml/XMLFileNestedConditionTest/testNestedCondition/'><strong>org.jboss.windup.rules.xml.XMLFileNestedConditionTest.testNestedCondition</strong></a></li><li><a href='http://10.3.8.5/job/windup-pr-builder/525/org.jboss.windup.rules.apps$windup-rules-xml-tests/testReport/org.jboss.windup.rules.xml/XMLHintsClassificationsTest/testHintAndClassificationOperation/'><strong>org.jboss.windup.rules.xml.XMLHintsClassificationsTest.testHintAndClassificationOperation</strong></a></li><li><a href='http://10.3.8.5/job/windup-pr-builder/525/org.jboss.windup.rules.apps$windup-rules-xml-tests/testReport/org.jboss.windup.rules.xml/XMLTransformationTest/testXSLTTransformation/'><strong>org.jboss.windup.rules.xml.XMLTransformationTest.testXSLTTransformation</strong></a></li><li><a href='http://10.3.8.5/job/windup-pr-builder/525/org.jboss.windup.rules.apps$windup-rules-xml-tests/testReport/org.jboss.windup.rules.xml/XMLTransformationXMLRulesTest/testXSLTTransformation/'><strong>org.jboss.windup.rules.xml.XMLTransformationXMLRulesTest.testXSLTTransformation</strong></a></li><li><a href='http://10.3.8.5/job/windup-pr-builder/525/org.jboss.windup.rules.apps$windup-rules-xml-tests/testReport/org.jboss.windup.rules.xml/XmlFileMultipleConditionTest/testNestedCondition/'><strong>org.jboss.windup.rules.xml.XmlFileMultipleConditionTest.testNestedCondition</strong></a></li><li><a href='http://10.3.8.5/job/windup-pr-builder/525/org.jboss.windup.rules.apps$windup-rules-xml-tests/testReport/org.jboss.windup.rules.xml/XmlFileParameterizedTest/testXmlParams/'><strong>org.jboss.windup.rules.xml.XmlFileParameterizedTest.testXmlParams</strong></a></li><li><a href='http://10.3.8.5/job/windup-pr-builder/525/org.jboss.windup.rules.apps$windup-rules-xml-tests/testReport/org.jboss.windup.rules.xml/XmlFileParameterizedTest/testXmlParamsDangling/'><strong>org.jboss.windup.rules.xml.XmlFileParameterizedTest.testXmlParamsDangling</strong></a></li><li><a href='http://10.3.8.5/job/windup-pr-builder/525/org.jboss.windup.rules.apps$windup-rules-xml-tests/testReport/org.jboss.windup.rules.xml/XmlFileParameterizedTest/testXmlParamsMultiCondition/'><strong>org.jboss.windup.rules.xml.XmlFileParameterizedTest.testXmlParamsMultiCondition</strong></a></li></ul><h3><a name='org.jboss.windup.tests:windup-tests' /><a href='http://10.3.8.5/job/windup-pr-builder/525/org.jboss.windup.tests$windup-tests/testReport'>org.jboss.windup.tests:windup-tests</a>: <span class='status-failure'>7</span></h3><ul><li><a href='http://10.3.8.5/job/windup-pr-builder/525/org.jboss.windup.tests$windup-tests/testReport/org.jboss.windup.tests.application/WindupArchitectureJEEExampleTest/testRunWindupJEEExampleMode/'><strong>org.jboss.windup.tests.application.WindupArchitectureJEEExampleTest.testRunWindupJEEExampleMode</strong></a></li><li><a href='http://10.3.8.5/job/windup-pr-builder/525/org.jboss.windup.tests$windup-tests/testReport/org.jboss.windup.tests.application/WindupArchitectureMediumBinaryModeTest/testRunWindupMedium/'><strong>org.jboss.windup.tests.application.WindupArchitectureMediumBinaryModeTest.testRunWindupMedium</strong></a></li><li><a href='http://10.3.8.5/job/windup-pr-builder/525/org.jboss.windup.tests$windup-tests/testReport/org.jboss.windup.tests.application/WindupArchitectureScanPackagesTest/testRunWindupScanPackages/'><strong>org.jboss.windup.tests.application.WindupArchitectureScanPackagesTest.testRunWindupScanPackages</strong></a></li><li><a href='http://10.3.8.5/job/windup-pr-builder/525/org.jboss.windup.tests$windup-tests/testReport/org.jboss.windup.tests.application/WindupArchitectureSeamBookingSourceTest/testRunWindupSeamBookingSourceMode/'><strong>org.jboss.windup.tests.application.WindupArchitectureSeamBookingSourceTest.testRunWindupSeamBookingSourceMode</strong></a></li><li><a href='http://10.3.8.5/job/windup-pr-builder/525/org.jboss.windup.tests$windup-tests/testReport/org.jboss.windup.tests.application/WindupArchitectureSmallBinaryMode2Test/testRunWindupTiny/'><strong>org.jboss.windup.tests.application.WindupArchitectureSmallBinaryMode2Test.testRunWindupTiny</strong></a></li><li><a href='http://10.3.8.5/job/windup-pr-builder/525/org.jboss.windup.tests$windup-tests/testReport/org.jboss.windup.tests.application/WindupArchitectureSourceModeTest/testRunWindupSourceMode/'><strong>org.jboss.windup.tests.application.WindupArchitectureSourceModeTest.testRunWindupSourceMode</strong></a></li><li><a href='http://10.3.8.5/job/windup-pr-builder/525/org.jboss.windup.tests$windup-tests/testReport/org.jboss.windup.tests.application/WindupArchitectureSpringSmallTest/testRunWindupSmallSpringApp/'><strong>org.jboss.windup.tests.application.WindupArchitectureSpringSmallTest.testRunWindupSmallSpringApp</strong></a></li></ul>
Test FAILed.

**Pull request merged**